src/contracts/option_round.cairo

```diff
@@ -1165,7 +1165,8 @@ mod OptionRound {
             ref self: ContractState, option_buyer: ContractAddress
         ) -> Result<u256, OptionRoundError> {
             //Check that the round is settled 
-            if (self.get_state() != OptionRoundState::Running) {
+            let state = self.get_state();
+            if (state != OptionRoundState::Running&&state!=OptionRoundState::Settled) {
```

Comment:
tokenizing can only happen while the round is running. open -> no options yet, auctioning -> no options yet, settled -> could just collect payout, only when running does it make sense to tokenize